ratis-server/src/test/java/org/apache/ratis/InstallSnapshotNotificationTests.java

```diff
@@ -332,6 +337,11 @@ private void testInstallSnapshotNotificationCount(CLUSTER cluster) throws Except
 
       // Make sure leader and followers are still up to date.
       for (RaftServer.Division follower : cluster.getFollowers()) {
+        long leaderIndex = leader.getRaftLog().getNextIndex();
+        // Give follower slightly time to catch up
+        RaftTestUtil.waitFor(
+                () -> leaderIndex == follower.getRaftLog().getNextIndex(),
```

Comment:
It should use leader.getRaftLog().getNextIndex() since the index may change.

Reply:
True, I fixed that in the last commit.